src/skas/core.py

```diff
@@ -10,7 +10,7 @@
 """
 Syllabification
 """
-nlp = spacy.load('es')
+nlp = spacy.load('es_core_news_md')
```

Comment:
Cannot be imported before being downloaded. The `setup.py` script might need to be updated as well.